### ISSUE TITLE
Houdini DSO error: Missing version information

### ISSUE THREAD

**Issue by ultra-sonic** [j-cube/multiverse#13]
Hey guys,

I am currently testing mv for houdini and get the following warning. It seems to work but maybe you could fix this just to be safe.

```
Houdini DSO error on '/localhost/repositories/3rd-party/multiverse/multiverseForHoudini-2.0.0-H15.0.244.16-Linux-x86_64/dso/libGABCMultiverse.so'
    Missing version information.

When creating custom operators with the HDK, remember to
#include <UT/UT_DSOVersion.h> to define the required version information.
```

keep up the great work!
Oliver

**Comment by pberto** [comment]
Hey Oliver, we actually do include `UT_DSOVersion.h` for every node. When do you get the warning?

Also which build of Houdini are you running? You should use `15.0.244.16` since plug-ins for Houdini need should be recompiled for each Houdini version.

Anyway will have a look next week.

Cheers.

**Comment by ultra-sonic** [comment]
Ah...snap...I am running 15.0.313 which was the latest production build a few days ago but now it is 15.0.347.
Is there a way to get a build for this one or should I re-install the old Houdini version (which I would dislike :)

**Comment by pberto** [comment]
No biggie, we can make a new build. Stay tuned.

**Comment by ultra-sonic** [comment]
Awesome! Standing by....
take your time!

**Comment by ultra-sonic** [comment]
any chance to see a more recent build against 15.0.393?

**Comment by pberto** [comment]
We will make a new free version release next week.

**Comment by ultra-sonic** [comment]
awesome!

**Comment by pberto** [comment]
Hello Olivier,

this is now released, compatible with 15.0.393. Note that it will work also on other H15, it will just complain about the version mismatch ;)

Have fun!

http://multi-verse.io/plugins/houdini/updates/
